java/build.gradle.kts

```diff
@@ -7,22 +7,50 @@ import com.google.protobuf.gradle.protoc
 plugins {
     id("momento.java-library-conventions")
 
-    kotlin("jvm")
     id("com.google.protobuf") version "0.8.16"
     idea
+    id("io.github.gradle-nexus.publish-plugin") version "1.3.0"
+    id("maven-publish")
+    id("signing")
+    id("ca.cutterslade.analyze") version "1.9.0"
 }
 
 val grpcProtobufVersion = "3.21.2"
 val grpcVersion = "1.47.0"
+val guavaVersion = "31.1-android"
 
 dependencies {
     implementation("io.grpc:grpc-protobuf:$grpcVersion")
+    implementation("com.google.api.grpc:proto-google-common-protos:2.0.1")
+
+    implementation("com.google.protobuf:protobuf-java:${grpcProtobufVersion}")
+    implementation("com.google.guava:guava:${guavaVersion}")
+
     implementation("io.grpc:grpc-stub:${grpcVersion}")
-    implementation("javax.annotation:javax.annotation-api:1.3.2")
+    implementation("io.grpc:grpc-api:${grpcVersion}")
```

Comment:
just curious why it looks like there are a bunch of new grpc deps coming in.  Are these just transitive deps from grpc-stub?

Reply:
Yeah. Those are all the formerly transitive dependencies being used.